wasdk/WebAssemblyStudio
# Fix various build issues (on Windows)

### Summary of Changes

* Fix bin files being non-executable through CRLF on Windows
* Fix invokation of bin files in package.json to be sure
* Upgrades webpack to v4 to(/and?) get rid of UglifyJS breaking the build
* Upgrades other dependencies as well to be sure

### Test Plan

* Run all the npm scripts on Windows
* Launch WebAssembly Studio and check that nothing broke

As a side note, when I navigate to https://webassembly.studio, there are no templates atm. Not sure if that's somehow connected. Works locally for me now, though.
